internal/service/clouduserteamassignment/resource_test.go

```diff
@@ -73,21 +98,53 @@ func configBasic(orgID, userID, teamName string) string {
 			org_id  = %[1]q  
 			team_id = mongodbatlas_team.test.team_id
 			user_id = %[2]q    
-		}  
+		} 
+		data "mongodbatlas_cloud_user_team_assignment" "test1" {
+			org_id   = %[1]q
+			team_id  = mongodbatlas_team.test.team_id
+			user_id  = mongodbatlas_cloud_user_team_assignment.test.user_id
+		}
+
+		data "mongodbatlas_cloud_user_team_assignment" "test2" {
+			org_id   = %[1]q
+			team_id  = mongodbatlas_team.test.team_id
+			username = mongodbatlas_cloud_user_team_assignment.test.username
+		}
 		`,
 		orgID, userID, teamName)
 }
 
+func configError(orgID, teamName string) string {
+	return fmt.Sprintf(`
+		resource "mongodbatlas_team" "test" {
+			org_id     = %[1]q
+			name       = %[2]q
+		}
+
+
+		data "mongodbatlas_cloud_user_team_assignment" "test" {
+		org_id  = %[1]q
+		team_id = mongodbatlas_team.test.team_id
+		}
+		`, orgID, teamName)
+}
+
 func checks(orgID, userID string) resource.TestCheckFunc {
 	return resource.ComposeAggregateTestCheckFunc(
 		resource.TestCheckResourceAttr(resourceName, "org_id", orgID),
 		resource.TestCheckResourceAttr(resourceName, "user_id", userID),
-
 		resource.TestCheckResourceAttrSet(resourceName, "username"),
 		resource.TestCheckResourceAttrWith(resourceName, "username", acc.IsUsername()),
 		resource.TestCheckResourceAttrWith(resourceName, "created_at", acc.IsTimestamp()),
-
 		resource.TestCheckResourceAttrWith(resourceName, "team_ids.#", acc.IntGreatThan(0)),
+
+		resource.TestCheckResourceAttr(dataSourceName1, "user_id", userID),
```

Comment:
```suggestion
		resource.TestCheckResourceAttr(dataSourceName1, "user_id", acc.IsUserID()),
```
Seeing IsUserID() definition, I assume this should be like this?

Reply:
`IsUserID()` was originally implemented for a test that has been removed, so this function is no longer needed.